extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -993,14 +1000,20 @@ export class AmpLightboxGallery extends AMP.BaseElement {
   // TODO (cathyxz): make this generalizable to more than just images
   enter_() {
     const sourceElement = this.getCurrentElement_().sourceElement;
-    return this.shouldAnimate_(sourceElement)
-        .then(shouldAnimate => {
-          if (shouldAnimate) {
-            return this.transitionIn_(sourceElement);
-          } else {
-            return this.fade_(0, 1);
-          }
-        });
+    if (!this.elementTypeCanBeAnimated_(sourceElement)) {
+      return this.fade_(0, 1);
+    }
+
+    const promises = [
+      this.shouldAnimate_(sourceElement),
+      this.getCurrentElement_().imageViewer.signals()
+          .whenSignal(CommonSignals.LOAD_END),
+    ];
+
+    return Promise.all(promises).then(values => {
+      return values[0] ? this.transitionIn_(sourceElement)
```

Comment:
What does this `values[0]` mean? Is it `shouldAnimate`? It would be easier to scan if it was like this:

```js
const shouldAnimate = values[0];
return shouldAnimate ? /* ... */;
```